kubernetes-cluster-eks/track.yml

```diff
@@ -0,0 +1,16 @@
+slug: kubernetes-cluster-eks
+id: 2avvhl0zzsxm
+type: track
+title: Amazon Elastic Kubernetes Service (EKS)
+teaser: A template track that includes a EKS cluster.
+description: |-
+  This template includes an EKS cluster and a workstation with the **kubectl** and **gcloud** commands preinstalled. Use it as a starting point for your own K8s enabled tracks.
+  Note - an EKS cluster takes up to 25 minutes to deploy. You may wish to utilize Hot Start to reduce startup times for your participants.
+icon: https://storage.googleapis.com/instruqt-frontend/img/tracks/default.png
+tags: []
+owner: instruqt
```

Comment:
Same comment as on your other PR, why not put these in the `templates` organization?